api/Window.json

```diff
@@ -4440,16 +4440,22 @@
             "edge": {
               "version_added": "≤79"
             },
-            "firefox": {
-              "version_added": "49",
-              "flags": [
-                {
-                  "type": "preference",
-                  "name": "dom.manifest.onappinstall",
-                  "value_to_set": "true"
-                }
-              ]
-            },
+            "firefox": [
+              {
+                "version_added": false
```

Comment:
This support statement isn't really necessary. The `"version_removed"` in the next statement has the same effect. I'd suggest removing this one.